pkg/cmdlets/auth/login.go

```diff
@@ -178,3 +182,27 @@ func promptToken(credentials *mono_models.Credentials, out output.Outputer, prom
 
 	return nil
 }
+
+// AuthenticateWithDevice attempts to authenticate this device with the Platform.
+func AuthenticateWithDevice(out output.Outputer) error {
+	deviceCode, err := model.RequestDeviceAuthorization()
```

Comment:
Nit: Bit miss-leading calling this `deviceCode` when it returns a struct with things like the verification URL.

Reply:
Okay.